src/OpenTelemetry.Exporter.Instana/OpenTelemetry.Exporter.Instana.csproj

```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <TargetFrameworks>netstandard2.0;net461</TargetFrameworks>
 	<PackageTags>Instana Tracing APM</PackageTags>
+	  <LangVersion>latest</LangVersion>
```

Comment:
Common lang version is defined under https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/9877a3220a3977dce2fb29d7dc2742dee553a0a8/build/Common.props#L3

I think, it will be fine to create separate PR with bumping it to 11.0.
Please revert change here.

Reply:
Ok, I will